packages/zosfiles/src/methods/list/List.ts

```diff
@@ -74,7 +74,28 @@ export class List {
 
             this.log.debug(`Endpoint: ${endpoint}`);
 
-            const response: any = await ZosmfRestClient.getExpectJSON(session, endpoint, reqHeaders);
+            let data = await ZosmfRestClient.getExpectString(session, endpoint, reqHeaders);
+            // Match each member name returned by list members endpoint
+            for (const match of data.matchAll(/"member":\s*"/g)) {
+                const memberMaxLength = 8;
+                let memberStartIdx = match.index + match[0].length;
+                // Loop through up to 8 characters that follow the opening quote
+                // eslint-disable-next-line @typescript-eslint/no-magic-numbers
+                for (let i = 1; i <= memberMaxLength; i++) {
+                    const memberChar = data.charCodeAt(memberStartIdx + i);
+                    // eslint-disable-next-line @typescript-eslint/no-magic-numbers
+                    if (memberChar < 32) {
+                        // Replace control characters with Unicode <?> symbol
+                        data = data.substring(0, memberStartIdx + i) + "\\ufffd" + data.substring(memberStartIdx + i + 1);
+                        memberStartIdx += 5;  // eslint-disable-line @typescript-eslint/no-magic-numbers
+                    // eslint-disable-next-line @typescript-eslint/no-magic-numbers
+                    } else if (memberChar === 34 && data.charCodeAt(memberStartIdx + i - 1) !== 92) {
```

Comment:
Can we add constants for these numbers? That may help devs understand what 34 and 92 are. :yum: